backend/internal/pkg/repositories/geocoding/geocoding.go

```diff
@@ -0,0 +1,31 @@
+package geocoding
+
+import "context"
+
+// An address, splitted into components
+type Address struct {
+	Street     string
+	City       string
+	State      string
+	PostalCode string
+	Country    string
+}
+
+// Geocoding result
+type Result struct {
+	// Address as resolved by the provider
+	Address Address
+	// The formatted form of the address above
+	FormattedAddress string
+	Latitude         float64
+	Longitude        float64
+	// The accuracy of the result, ranging from 0 to 1.
```

Comment:
Is this comment accurate?  In our service, we have a condition to check if accuracy < 1 that denotes inaccurate result.